build.gradle

```diff
@@ -23,12 +23,33 @@ repositories {
 	mavenCentral()
 }
 
-
 dependencies {
 //	implementation('org.springframework.boot:spring-boot-starter-data-jpa')
 	implementation('org.springframework.boot:spring-boot-starter-websocket')
 //	implementation('org.flywaydb:flyway-core')
 //	runtimeOnly('org.postgresql:postgresql')
 	compileOnly('org.projectlombok:lombok')
 	testImplementation('org.springframework.boot:spring-boot-starter-test')
+    testCompile('org.assertj:assertj-core:3.11.1')
+    testCompile('io.cucumber:cucumber-java8:4.0.0')
+    testCompile('io.cucumber:cucumber-junit:4.0.0')
```

Comment:
I'd rather see a separate `featureTest` source set and associated configurations in Gradle, rather than lumping the Cucmber features and fixtures in the same source tree as unit tests.